contracts/ckb-bitcoin-spv-type-lock/src/operations/reorg.rs

```diff
@@ -53,13 +53,17 @@ pub(crate) fn reorg_clients(inputs: &[usize], outputs: &[usize], script_hash: &[
     let (output_client, output_info_index) =
         load_outputs(outputs, &expected_info, expected_client_ids)?;
     {
-        let new_chain_work: U256 = output_client
-            .headers_mmr_root()
-            .partial_chain_work()
-            .unpack();
-        if previous_chain_work >= new_chain_work {
-            return Err(InternalError::ReorgNotBetterChain.into());
-        }
+        // If there is a limit on the number of headers to update
```

Comment:
This check protected the cell from being modified by malicious operators.

IMO we should only remove this check from testnet and keep it on the mainnet.

https://github.com/utxostack/ckb-bitcoin-spv/blob/master/verifier/src/utilities/bitcoin.rs#L51

Reply:
done